app/posts/register-trainee-teachers/2022-05-13-showing-providers-the-funding-they-receive.md

```diff
@@ -0,0 +1,108 @@
+
+DTTP shows providers the money they receives from the DfE for bursaries, grants and scholarships. In research we saw that this information is useful, but not as complete or as clear as users would like.
+
+The value it offers users is providing confirmation to users they’ve registered their trainees correctly and that the DfE will pay as expected.
+
+We saw that users want more transparency and clarity in how the DfE decides how much it pays each month. There is rarely a straightforward relationship between the number of trainees eligible for funding and what the provider gets paid. The first 3 month’s payments based on the number of trainees given unconditional offers in Apply (minus 25% to allow for trainees that do not start). After these first 3 payments, the remaining payments will correct over or under payments. The funding team update their payment models quarterly. If a provider’s trainees change status this will only be reflected at the end of that quarter. This adds to the need for more over or under payment, increasing the complexity for providers in unpicking why they have been paid what they’ve been paid.
+
+As the simplest way to show this information, we are adding the same funding data to Register. The data will continue to be uploaded from spreadsheets to Register.
+
+## How DTTP shows the funding information
+
+The funding team break payments into 2 sets of data. Accredited providers (HEIs and SCITTs) and lead schools get slightly different versions of the 2 sets.
+
+### 1. The ‘payment profile’
+
+This is the money the provider receives each month, separated by types of payment (for example, total amount of bursary money for all trainees for that month).
+
+The ‘payment profile’ always covers the whole academic year. It has payments that have been made and predicted payments. For example, if the user is looking at their payment profile in August they see one payment made in August and 11 predicted payments for the rest of the year. If the user is looking at their payment profile in July they see 12 payments that have been made and 6 predicted payments. More payment types are added through the year.
+
+DTTP adds each month’s payment profile data to the last one. For example, in August a user can see one payment profile, in July the user can see 12 payment profiles. Each payment profile repeats the data on payments made in previous months, but updates the current months and the predicted payments.
+
+### 2. The ‘training bursary summary’ or ‘school direct summary’
+
+Accredited providers (HEIs and SCITTs) see a list of the bursary and scholarship amounts their trainees receive.
+
+Lead schools see the grants they receive.
+
+These summaries are based on the October census. They're not updated throughout the year. For example, if a trainee withdraws the amount shown is not adjusted. Because it’s based on the October census, the summary is not available until December.
```

Comment:
```suggestion
These summaries are based on the October census. They’re not updated throughout the year. For example, if a trainee withdraws then the amount shown is not adjusted. Because it’s based on the October census, the summary is not available until December.
```